### PR TITLE
Docs: make it clearer node_modules is excluded, and ! can be used to …

### PR DESCRIPTION
**Pull request by hzoo**
…include

Maybe we don't need to do this but it may be clearer like this?

### CODE REVIEW

**Comment by zxqfox** [comment]
:+1: from me.

@markelog `#Default` block looks like inconsistency ;-( What you think?

**Comment by hzoo** [comment]
Yeah I think so. Showing it like `"excludeFiles": ["node_modules/**"]` was kind of weird since it makes it sound like you still have to do it yourself?

File: OVERVIEW.md [hunk 1]
```diff
@@ -252,16 +252,14 @@ Type: `Array`
 
 Values: Array of file matching patterns
 
-#### Example
+> `node_modules` is excluded by default
```
Comment (markelog) [inline OVERVIEW.md]: The next string would also be included in the quote block

**Comment by markelog** [comment]
@zxqfox 
You mean in `fileExtensions`? Yeah, i think we should be consistent

File: OVERVIEW.md (hunk 1 above)
Comment (hzoo) [inline OVERVIEW.md]: Yeah I was just going to include both in the quote

**Comment by hzoo** [comment]
DId you want me to change something with this?

**Comment by markelog** [comment]
It seems now we have inconsistency of how we document `fileExtension` and `excludeFiles`?

File: OVERVIEW.md (hunk 1 above)
Comment (markelog) [inline OVERVIEW.md]: Why this is a quote?

**Comment by hzoo** [comment]
Ok updated

**Comment by markelog** [comment]
LGTM

**Pull request closed without merging**